### PR TITLE
Use separated proto repository

### PR DESCRIPTION
**Pull request by im-kulikov**
Workflow:
- update goland dependencies
- remove old `*.proto` and `*.pb.go` files
- download specific version of neofs-proto (will be published soon)
- extract proto files into `vendor/proto`
- walk throw `vendor/proto` folders that contains `*.proto` files and override in repo

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,38 @@
...
+
+# Dependencies
+deps:
+	@echo "${B}${G}=> Golang modules ${R}"
+	@go mod tidy -v
+	@go mod vendor
+
+	@echo "${B}${G}=> Cleanup old files ${R}"
+	@for f in `find . -type f -name '*.pb.go' -not -path './vendor/*' `; do \
+		rm $$f; \
```
Comment (fyrchik) [inline Makefile]: It is probably better to provide `-exec rm {} \;` argument to find than use for loop.

Comment (im-kulikov) [reply Makefile]: kk

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,34 @@
...
+	@echo "${B}${G}=> Cleanup old files ${R}"
+	@find . -type f -name '*.pb.go' -not -path './vendor/*' -exec rm {} \;
+	@find . -type f -name '*.proto' -not -path './vendor/*' -not -name '*_test.proto' -exec rm {} \;
+
+	@echo "${B}${G}=> NeoFS Proto files ${R}"
+	@mkdir -p ./vendor/proto
+	@curl -sL -o ./vendor/proto.tar.gz $(PROTO_URL)
+	@tar -xzf ./vendor/proto.tar.gz --strip-components 1 -C ./vendor/proto
+	@for f in `find ./vendor/proto -type f -name '*.proto' -exec dirname {} \; | sort -u `; do \
+		cp $$f/*.proto ./$$(basename $$f); \
```
Comment (fyrchik) [inline Makefile]: It won't work if there are spaces in a `basename $f`.

Comment (im-kulikov) [reply Makefile]: there are no spaces expected

**Pull request merged**